doc/install_ubuntu.txt

```diff
@@ -49,6 +49,35 @@ These instructions were written for Ubuntu 11.04, 11.10, 12.04, 12.10, 13.04,
 Installation steps
 ~~~~~~~~~~~~~~~~~~
 
+Ubuntu 14.04:
+
+The following will set up Theano from source packages in a Virtual Environment
+on an Ubuntu 14.04 machine.
+
+.. code-block: bash
+
+    sudo apt-get install python-numpy python-scipy python-dev python-pip \
+    python-nose g++ libopenblas-dev git libblas-dev liblapack-dev fortran-compiler
+    # Configure openblas as our BLAS of record...
+    update-alternatives --list libblas.so.3 | grep openblas
+    sudo update-alternatives --set libblas.so.3 $(update-alternatives --list libblas.so.3 | grep openblas)
+    virtualenv -p python2.7 theano-env
+    source theano-env/bin/activate
+    # Theano setup.py won't pull in Numpy, so 
+    # scipy dependency installation will fail
+    pip install numpy
+    
+    # Bleeding edge installation (Theano from git):
```

Comment:
I would add more distinction to the two install blocks.  Like break out of the ::bash block and make the comment text on the page.  Otherwise it's too easy for people to think they have to somehow install theano twice.